apps/docs/src/app/core/component-docs/link/examples/link-example.component.ts

```diff
@@ -1,7 +1,25 @@
-import { Component } from '@angular/core';
+import { Component, OnInit } from '@angular/core';
+import { BehaviorSubject } from 'rxjs';
+import { RtlService } from '@fundamental-ngx/core';
 
 @Component({
     selector: 'fd-link-example',
     templateUrl: './link-example.component.html',
 })
-export class LinkExampleComponent {}
+export class LinkExampleComponent implements OnInit {
+
+    arrowRight$: BehaviorSubject<string> = new BehaviorSubject<string>('slim-arrow-right');
+    arrowLeft$: BehaviorSubject<string> = new BehaviorSubject<string>('slim-arrow-left');
+
+    constructor(private rtlService: RtlService) { }
+
+    ngOnInit(): void {
+        if (this.rtlService) {
+            this.rtlService.rtl.subscribe(value => {
```

Comment:
Need to unsubscribe and can be simplified to Observable.

Reply:
You can skip rtlService null check since all examples are handled by us